### PR TITLE
update faq.md (fixes #83)

### LINKED ISSUE #83: Host key verification failed error
This is error is prompted during the ssh login attempt after the RSP4 booted with a new tree houses image. I was using PC as ssh client. 

User won't be able to login to the RP4. The following command will fail to execute
`ssh -v pi@192.168.2.1`

### PR DESCRIPTION
**Pull request by phamduchongan93**
I separated the pulls and make them branched out from the master. Please verify the integrity of files. Thanks for being patient with my PRs. This PR is meant to solve #83 issue.
Raw Githack [link](https://raw.githack.com/phamduchongan93/phamduchongan93.github.io/faq_with_question/#!pages/vi/faq.md)

### CODE REVIEW

**Comment by dogi** [comment]
please add the raw githack link

File: pages/vi/faq.md [hunk 1]
```diff
@@ -13,6 +13,7 @@
 - **Q9:** [How long do I have to complete these steps?](#Q9:_How_long_do_I_have_to_complete_these_steps?)
 
 ## Technical Questions
+- **Q1:** [What if you encounter *Host key verification fail* or *Add correct host key in C:\\Users\\USER/.ssh/known_hosts to get rid of this message*?](#Q1:_What_if_you_encounter_Host_key_verification_fail?)
```
Comment (dogi) [inline pages/vi/faq.md]: Q11?

**Review by dogi — CHANGES_REQUESTED** [review]

File: pages/vi/faq.md (hunk 1 above)
Comment (phamduchongan93) [reply pages/vi/faq.md]: I was intending to add it in technical question.

Comment (dogi) [reply pages/vi/faq.md]: yes
we wanna also keep them unique by number ;)
so that there is no misunderstanding

File: pages/vi/faq.md [hunk 2]
```diff
@@ -66,6 +67,20 @@
     - Name the organization '&lt;YourUserName&gt;-OLE', select 'My personal account', choose the 'Team For Open Source' option, and enter your email address to create the account.
     - Now it will prompt you for which profile to use when you fork the [treehouses repository](https://github.com/treehouses/treehouses.github.io).
 
+#### Q11: What if you encounter Host key verification fail?
+
++ To fix the issue, please type  `rm YourHomeDirectory/.ssh/known_hosts` in your powershell or terminal.
```
Comment (dogi) [inline pages/vi/faq.md]: in the 10 steps we support only `bash` driven shells
- macOs: console (...)
- windows: git bash (cygwin, wsl and wsl2 with debian)
- ubuntu: terminal (konsole, ... )

Comment (dogi) [reply pages/vi/faq.md]: so please reduce the message to only terminal syntax ;)

File: pages/vi/faq.md [hunk 3]
```diff
@@ -66,6 +67,20 @@
...
+For example: I would type
+
+ *In linux (ubuntu, fedora, ... )*  
+ 
+ `rm "/home/$(whoami)/.ssh/known_hosts"`
+       
+*In Windows*
+
+ `rm C:\Users\"your_user_name"\.ssh\known_hosts`
+
```
Comment (dogi) [inline pages/vi/faq.md:76]: also there are at least two more options
this one here I would call the nuclear one ;)

for more info look at https://linuxcommando.blogspot.com/2008/10/how-to-disable-ssh-host-key-checking.html

**Review by dogi — CHANGES_REQUESTED** [review]

**Review by dogi — APPROVED** [review]

**Pull request merged**